Gruntfile.js

```diff
@@ -99,6 +99,9 @@ module.exports = function(grunt) {
 			files: buildFiles.map( function( file ) {
 				return BUILD_DIR + file;
 			}),
+			blocks: [
```

Comment:
I'm wondering whether this clean task is necessary. We didn't have it before and it was working properly.

Reply:
I was just looking into this actually. I just ran a test with that clean and the blocks folder was deleted and I ran again and the folder was preserved. There seems to be an issue here though. I'm seeing that the `{blockname}.php` is being created as well as the `{blockname}/index.php`

Reply:
Removed that file. Seems like those extra files came over with `copy:files` I update this PR with some more changes. We can update the task with the new diff if it passes

Reply:
Cool, Travis is happy now. So it looks like unit tests depend on the existence of those files. It's unclear to me why they get removed in the first place. Regardless, it follows what was here before so I can look into it separately.

Reply:
> It's unclear to me why they get removed in the first place

I think the `clean:blocks` step was executing after the copy steps. So it would copy then delete them.